tests/Unit/Command/FetchAssetsCommandTest.php

```diff
@@ -138,7 +138,7 @@ public function executeFallsBackToLatestAssetsIfAssetsCannotBeDownloadedAndFails
     /**
      * @test
      */
-    public function executeFailsIfAssetsAreAlreadyDownloaded(): void
+    public function executeWarnsIfAssetsAreAlreadyDownloaded(): void
```

Comment:
From what I can see, those test cases all test the same behavior:

* `executeWarnsIfAssetsAreAlreadyDownloaded`
* `executeSucceedsWithWarningsIfAssetsAreAlreadyDownloaded`
* `executeWarningsIfAssetsAreAlreadyDownloaded`

Please remove two of them.

Reply:
Wow, that's not at all embarrassing. I've removed the duplicates in the most recent changes. Apologies.